scsi_defs.h

```diff
@@ -18,6 +18,7 @@
 #define READ_DISC_INFORMATION           0x51
 #define MODE_SELECT_10                  0x55
 #define MODE_SENSE_10                   0x5a
+#define RECEIVE_COPY_RESULTS            0x84
```

Comment:
I think spc4. I do not think spc5 has been finalized yet. This shows

http://www.t10.org/members/w_spc5.htm

in devel still.